plugins/ssh-plugin/src/util/util.ts

```diff
@@ -24,7 +24,7 @@ export async function askHostName(message?: string): Promise<string | undefined>
     placeHolder: message ? message : 'Please provide a hostname e.g. github.com',
     validateInput: (text: string) => {
       if (!hostNamePattern.test(text)) {
-        return 'Invalid hostname';
+        return Promise.resolve('Invalid hostname');
```

Comment:
as it's an async method, why do we need to use a Promise there ?

Reply:
<img width="1155" alt="promise_error" src="https://user-images.githubusercontent.com/5676062/126462960-5464717e-5dd8-49bf-abdc-c64e07432f5f.png">

Reply:
ok probably then async on the validateInput arrow function

```
validateInput: async (text: string) => {
if (!hostNamePattern.test(text)) {
        return 'Invalid hostname';
```
but it's only a suggestion, you can merge it without that

Reply:
I have merged the PR as is to speed up the alignment with upstream.
will check it and provide a separate improvement for this place.
thanks!

Reply:
provided a separate PR https://github.com/eclipse-che/che-theia/pull/1188